src/pages/Detailpage/DetailComponents/Rate/Rate.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react';
+
+const Rate = () => {
+  const [isLiked, setIsLiked] = useState(true);
+  const isLikeClick = () => {
+    setIsLiked(!isLiked);
+  };
+
+  return (
+    <button className="starBtn" onClick={() => isLikeClick()}>
```

Comment:
```suggestion
    <button className="starBtn" onClick={isLikeClick}>
```
굳이 새로운 함수 하나 더 만들지말고 함수 그대로 이벤트리스너로 추가해줘도 되보이네요!

Reply:
반영완료